src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -3985,6 +4082,109 @@ namespace ZoneTempPredictorCorrector {
 				SNLoad = 0.0;
 			}
 
+			//Hybrid modeling start: Added by Sang Hoon Lee May 2015
+			if (( HybridModelZone( ZoneNum ).InternalThermalMassCalc || HybridModelZone( ZoneNum ).InfiltrationCalc ) && !WarmupFlag && !DoingSizing ) {
```

Comment:
This if statement will be hit every timestep and iteration - would be a little faster to wrap this all with a higher-level if statement with the simple global flag instead of grabbing two values from the HybridModelZone array every time.
if ( FlagHybridModel )